### ISSUE TITLE
Negative samples for detection

### ISSUE THREAD

**Issue by themantalope** [Project-MONAI/tutorials#1194]
With the retinanet model and the current implementation and loss function is it possible to train with images that contain no objects (bounding boxes)? Or does the loss function expect that for each image there will be at least one box?
